Capstone/ViewControllers/MapViewController.m

```diff
@@ -202,6 +227,15 @@ -(NSString *) setDate:(NSDate *)date {
     return stringDate;
 }
 
+- (BOOL)isSameMonth:(NSDate*)date1 otherDay:(NSDate*)date2 {
+    NSCalendar* calendar = [NSCalendar currentCalendar];
+    unsigned unitFlags = NSCalendarUnitYear | NSCalendarUnitMonth |  NSCalendarUnitDay;
+    NSDateComponents* comp1 = [calendar components:unitFlags fromDate:date1];
+    NSDateComponents* comp2 = [calendar components:unitFlags fromDate:date2];
+    return [comp1 month] == [comp2 month] &&
+    [comp1 year]  == [comp2 year];
+}
+
```

Comment:
it seems repetitive to the method in "[CalendarViewController.m]", can we reuse the method?